### PR TITLE
New way to use a strategy

### PR DESCRIPTION
**Pull request by marioarranzr**
I found a bit complicated to copy and paste the example into `main.go` and I think with this is a simpler solution to switch strategies.

Congrats, you are doing a really good job!

### CODE REVIEW

**Comment by saniales** [comment]
Can you summarize what you did? 
Can you cherry pick your commit into the top of `develop` branch and open a new pr? This would be awesome
Thanks and let's keep rockin

**Comment by saniales** [comment]
I will close this when @marioarranzr comments

**Comment by marioarranzr** [comment]
Sorry, it was closed automatically when I removed the branch to create a new one at the top of development. I summarise in the new PR
